CMakeLists.txt

```diff
@@ -277,6 +277,7 @@ add_subdirectory(${LIBRARIES}/utils)
 add_subdirectory(${FILAMENT}/filament)
 add_subdirectory(${FILAMENT}/shaders)
 add_subdirectory(${EXTERNAL}/robin-map/tnt)
+add_subdirectory(${EXTERNAL}/smol-v/tnt)
```

Comment:
It had to be moved here to appease iOS, which doesn't use the bigger list of dependencies.  Note that this builds SMOL for all platforms, but SMOL doesn't actually get *used* on all platforms (e.g. it is excluded from the Android GL-only config). By doing it this way, we can prep for the future and avoid creating a new conditional just for iOS.